public/index.php

```diff
@@ -3,38 +3,29 @@
 declare(strict_types=1);
 
 use App\Kernel;
-use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
-require __DIR__.'/../vendor/autoload.php';
-
-// The check is to ensure we don't use .env in production
-if (! isset($_SERVER['APP_ENV'])) {
-    if (! class_exists(Dotenv::class)) {
-        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
-    }
-    (new Dotenv())->load(__DIR__.'/../.env');
-}
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
Any reason for using a different coding style than the Symfony ones? There shouldn't be spaces between concatenation within Symfony guidelines. Might wanna check on that for the entire PR